conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/ForkDetector.scala

```diff
@@ -0,0 +1,117 @@
+package tech.cryptonomic.conseil.indexer
+
+import tech.cryptonomic.conseil.indexer.ForkDetector._
+
+import cats._
+import cats.implicits._
+import com.typesafe.scalalogging.LazyLogging
+
+/** Definitions and common utilities for fork-detection */
+object ForkDetector {
+
+  /** A data type that express the of search results of a comparison by level */
```

Comment:
This comment is not clear.

Reply:
That looked like some half-backed doc... I tried to expand and clarify